### DISCUSSION TITLE
Start browser from mitmdump?

### DISCUSSION THREAD

**Discussion by redraw** [mitmproxy/mitmproxy#5179] [Ideas]
Hi! is there any way to start the proxied browser using `mitmdump`?
It could be a single flag
 
```
mitmdump --browser-start
```

or maybe it could accept commands (like currently accept scripts) at launch time like,

```
mitmdump --command "browser.start"
```

**Comment by mhils** [comment]
I don't think there is, but running a command after starting mitmdump sounds like a useful feature! For the moment you can approximate it with a custom addon:

```python
from mitmproxy import ctx

def running():
    ctx.master.commands.call("browser.start")
```

**Reply by redraw** [reply]
thanks! I wonder why it opens two browser (chrome) instances that way

**Reply by mhils** [reply]
I think that's a bug that is fixed on main. 🙈

**Reply by redraw** [reply]
+1 I'll test against `main`

**Reply by redraw** [reply]
great! worked.

did `pipx install git+https://github.com/mitmproxy/mitmproxy`
